src/components/Footers/FooterBase/index.tsx

```diff
@@ -0,0 +1,30 @@
+import BoxBase from "src/components/Boxs/BoxBase";
+import TypographyBase from "src/components/Typography/TypographyBase";
+
+export interface FooterBaseProps {
+    bgcolor?: "primary" | "secondary";
+}
+
+const FooterBase = (props: FooterBaseProps) => {

```

Comment:
Cool, but this is not a material component so I think we should not have "base" in its name

Reply:
oh that's the reason why components have "Base" in their name :))